src/app/utils/history/change.ts

```diff
@@ -1,9 +1,11 @@
 import { Manuscript } from 'app/types/manuscript';
+import { JSONObject } from 'app/types/utility.types';
 
 export abstract class Change {
```

Comment:
I think it would be wise to consider versioning this, so that changes contain a version number of sorts. This should help us cope with for example a case where maybe in the future we update the changes object or rename a property, and then open a really old article in the editor with some changes saved on the server in the old format?